runtime/js/90_deno_ns.js

```diff
@@ -96,8 +96,22 @@ const denoNs = {
   iter: io.iter,
   iterSync: io.iterSync,
   SeekMode: io.SeekMode,
-  read: io.read,
-  readSync: io.readSync,
+  read(rid, buffer) {
```

Comment:
Doing this hear minimises the impact of the printed alert on unintended APIs.

Reply:
What do you mean? Are there internal APIs that call this function?

Reply:
Other APIs call the underlying `read()`. E.g. `Deno.stdin.read()` calls that `read()`. In other words, this change purely affects `Deno.read()`. This is the case for other similar APIs.

Not just that, we centralise where all these `warnOnDeprecateApi()` calls are.